### PR TITLE
Add PHP example

### PR DESCRIPTION
**Pull request by keitwb**

### CODE REVIEW

File: php/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+**If you do not have the [Smart Agent deployed on the same
+host](https://docs.signalfx.com/en/latest/apm/apm-deployment/smart-agent.html)**,
+you can configure the PHP tracer to send directly to our global ingest
+server by setting the following envvars in your shell before running the next
+command:
+
+```sh
+$ export SIGNALFX_ACCESS_TOKEN=<your org token>
+$ export SIGNALFX_ENDPOINT_URL=https://ingest.signalfx.com/v1/trace  # Or the appropriate ingest for your realm
```
Comment (rmfitzpatrick) [inline php/README.md]: These should probably be standardized https://github.com/signalfx/signalfx-ruby-tracing/blob/master/lib/signalfx/tracing.rb#L17

**Comment by mpetazzoni** [comment]
I have no idea what to review in here. Could you break this up in more manageable commits so reviewers can rightfully ignore commits that just "import" a bunch of 3rd party code without fear of missing anything in their review? Thanks!

File: php/README.md (hunk 1 above)
Comment (keitwb) [reply php/README.md]: Probably so, but I don't think the Ruby name of "ingest" is the best.  The Jaeger "endpoint" term seems to be the most generic and clear to cover everything from going to the agent, gateway, and directly to ingest.

**Comment by keitwb** [comment]
@mpetazzoni The first six files are the core of it, the rest are just dependencies.  Github seems to have gotten pretty smart about ordering files but I'll split it up into two commits.

**Comment by rmfitzpatrick** [comment]
Closing in lieu of https://github.com/signalfx/tracing-examples/pull/41.  Will reopen and update with Symfony support.

**Pull request closed without merging**